content/300-guides/050-database/880-supabase.mdx

```diff
@@ -28,6 +28,28 @@ Many aspects of using Prisma with Supabase are just like using Prisma with any o
 - use [`db push`](/concepts/components/prisma-migrate/db-push) to push changes in your schema to Supabase
 - use [Prisma Client](/concepts/components/prisma-client) in your application to talk to the database server at Supabase
 
+## Specific considerations
```

Comment:
I think on our side, we could "widen" the description here a bit form the specific use case (as Supabase does it):
If you want to use the database connection with pooling enabled that Supabase offers, these are the things to do.
(Of course we can also mention the use case serverless as the reason for actually wanting to do so - but I guess many users do so in any case as the URL pretty prominent in the Supabase UI)